examples/jsm/nodes/accessors/NormalNode.js

```diff
@@ -41,6 +42,17 @@ NormalNode.prototype.generate = function ( builder, output ) {
 
 			break;
 
+		case NormalNode.BENT:
+
+			if ( builder.isShader( 'fragment' ) ) {
+				if( builder.context.anisotropy ) result = 'getBentNormal( geometry, anisotropyFactor, roughnessFactor )';
```

Comment:
Some code style feedback:
```js
if ( builder.context.anisotropy ) {

     result = 'getBentNormal( geometry, anisotropyFactor, roughnessFactor )';

} else {

    result = 'geometry.normal';

}
```
Also ensure that any kind of block is padded by blank lines.

Reply:
@DanielSturk @Mugen87 I proposed some improvements here https://github.com/DanielSturk/three.js/pull/1 for your PR related to NodeMaterial.